install/kubernetes/cilium/values.yaml

```diff
@@ -428,7 +428,7 @@ certgen:
   image:
     repository: docker.io/cilium/certgen
     tag: v0.1.3@sha256:460fdcbabfa08b435dbee455e88add1ec54761ca2693e1d9885408ff55ed7f8e
-    pullPolicy: IfNotPresent
+    pullPolicy: Always
```

Comment:
nit: version + sha256 combination seems unique to me, can you explain why this got changed to always ?

Reply:
`Always` is the pullPolicy by default on the latest version of the tree, we only change it to `IfNotPresent` when we create branch versions or release versions.